index.html

```diff
@@ -0,0 +1,39 @@
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Delia Cercel Portfolio Site</title>
+        <!-- Link to stylesheet -->
+        <link href="stylesheets.css/style.css" rel="stylesheet"/>
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans|Roboto" rel="stylesheet">
+    </head>
+    <body>
+        <main>
+            <div class="flex-column-wrapper">          
+                <header class="left">
+                    <div class="profile-picture">
+                        <img src="images/profile.jpg" alt="profile picture" width="300"/>
+                    </div>
+                    <h1>Delia Cercel</h1>
+                </header>
+                <nav>
+                    <ul class="social-links">
+                        <li class="current-page">Home</li>
+                        <li><a href="./about.html">About</a></li>
+                        <li><a href="./portfolio.html">Portfolio</a></li>
+                        <li><a href="./hobby-blog.html">Hobbies</a></li>
+                        <li><a href="https://github.com/Parseluni">Github</a></li>
+                        <li><a href="https://www.linkedin.com/in/delia-cercel-282b052a/">LinkedIn</a></li>
```

Comment:
```suggestion
                        <li><a href="https://github.com/Parseluni" target="_blank">Github</a></li>
                        <li><a href="https://www.linkedin.com/in/delia-cercel-282b052a/" target="_blank">LinkedIn</a></li>
```

Adding `target="_blank"` will open the link on a new tab which provides a better user experience